Add instructions for local postgres

sjwmoveon opened the pull request:
## Description
Add additional documentation on how to set up Postgres if using a local installation and not Docker.  I tried setting up locally with a preexisting Postgres installation and found I needed to make the listed changes.

# Checklist:

- [N/A] I have manually tested my changes on desktop and mobile
- [N/A] The test suite passes locally with my changes
- [N/A] If my change is a UI change, I have attached a screenshot to the description section of this pull request
- [X] [My change is 300 lines of code or less](https://github.com/StateVoicesNational/Spoke/blob/main/CONTRIBUTING.md#submitting-your-pull-request), or has a documented reason in the description why it’s longer
- [X] I have made any necessary changes to the documentation
- [N/A] I have added tests that prove my fix is effective or that my feature works
- [N/A] My PR is labeled [WIP] if it is in progress

sjwmoveon committed:
Add instructions for local postgres

Add additional documentation on how to set up Postgres if using a local installation and not Docker.

engelhartrueben commented on docs/HOWTO_USE_POSTGRESQL.md:
```diff
@@ -3,9 +3,19 @@
+     * Grant permissions to the new Spoke user:
+       * `psql -d spokedev -c "GRANT ALL PRIVILEGES ON DATABASE spokedev TO spoke;"`
+       * `psql -d spokedev -c "GRANT ALL PRIVILEGES ON schema public TO spoke;"`
+     * Also run the commands from `./dev-tools/create-test-database`:
```
This tool won't work for local pg because its set up for docker.

sjwmoveon replied:
Right, I'm not saying to run the script directly but to manually run the commands to create a test database instead.

engelhartrueben replied:
Ahh I see. I would instead have you place those instructions directly in the document (minus the docker stuff). I can see new devs getting confused like I did.

sjwmoveon replied:
I updated the explanation, hopefully this is more clear!

sjwmoveon committed:
Clarify test database commands

We don't actually want to run the script, which is built for Docker environments - we just want to run the SQL commands from that script.

engelhartrueben reviewed (approved):
🎉
